patches/typescript-react-function-component-props-handler+1.1.1.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/node_modules/typescript-react-function-component-props-handler/index.js b/node_modules/typescript-react-function-component-props-handler/index.js
+index 5c636b9..ca4c9a0 100644
+--- a/node_modules/typescript-react-function-component-props-handler/index.js
++++ b/node_modules/typescript-react-function-component-props-handler/index.js
+@@ -47,10 +47,6 @@ function checkForProptypes(path, paramTypeName) {
+ }
+ 
+ function setParamsTypeDefinitionFromFunctionType(documentation, path) {
+-    if (path.parentPath.node.init && path.parentPath.node.init.params.length === 0) {
```

Comment:
Great work overall 👏!
For our case I guess it works fine, but as for the external library PR this check might be crucial in some cases. Perhaps this would work better:
```suggestion
+ if (path.parentPath.node.init  && Array.isArray(path.parentPath.node.init.params) && path.parentPath.node.init.params.length === 0) {
        return;
    }
```

Reply:
This is true, I had this thought too as regards the check, there must be a reason for it and maybe not in our favor, but in favor of other users. I think this is a better check. 
Thanks! 
updated [here](https://github.com/commercetools/ui-kit/pull/2219/commits/1ea9a91c0663deddc79a051a926aa818db13d458)